.github/workflows/formatting-linting.yml

```diff
@@ -25,6 +25,12 @@ jobs:
 
       - name: 'Lint subgraph'
         run: 'yarn lint:subgraph:ts >> $GITHUB_STEP_SUMMARY'
+      
+      - name: 'Install js-client dependencies'
```

Comment:
why do you need the js-client dependencies when you only lint?

Reply:
Solved by merging in develop.

Reply:
Resolved by now